First editing pass through troubleshooting guide (#17154)
metabase/metabase: docs/troubleshooting-guide/timezones.md

A few specific problems are described below. If none of them apply, please [open a bug report][metabase-file-bug] with the above information (time zones and the results of the second troubleshooting process) as well as your Metabase, OS, and web browser versions.

## Specific Problems

### SQL queries are not respecting the Reporting Time Zone setting

**How to detect this:**
You are not able to click on a cell in a result table or a chart.

**How to fix this:**
We do not currently apply a reporting time zone to the results of SQL queries, so you should set one manually.

### Dates without an explicit time zone are being converted to another day

**How to detect this:**
This occurs when you are grouping by a date (rather than by a time) that does not have a time zone attached to it. Look at every time field your question uses in the Data Model Reference, and see if any of them are simply a "Date" field.

**How to fix this:**

> Make sure the server time zone reflects the reporting time zone, because when a query is run on Metabase the server applies the time zone to which it is set to that date.

Reviewer: ```suggestion
Make sure the server time zone reflects the reporting time zone, because when a query is run on Metabase, the server applies the configured time zone to that date.
```
